### PR TITLE
feat: Add support for queueMicrotask (Closes #13)

### LINKED ISSUE #13: Support queueMicrotask() as soon as it lands in first engines
Announcement: https://twitter.com/domenic/status/1024774932453093376

Spec: https://html.spec.whatwg.org/multipage/timers-and-user-prompts.html#microtask-queuing

Let's add support as soon as first engine implements it.

### PR DESCRIPTION
**Pull request by Richienb**
Closes #13

cc @medikoo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,8 @@
 To be used in environment agnostic modules that need nextTick functionality.
 
 - When run in Node.js `process.nextTick` is used
-- In modern browsers microtask resolution is guaranteed by `MutationObserver`
+- In modern browsers, microtask resolution is guaranteed by `queueMicrotask`
```
Comment (medikoo) [inline README.md]: Let's rename _browsers_ to _engines_ (technically non-browser engines may also implement it, e.g. Node.js does, altough we stick to `process.nextTick` there)

Comment (Richienb) [reply README.md]: @medikoo Done!

**Comment by Richienb** [comment]
@medikoo Please re-review.

**Comment by medikoo** [comment]
Published with v1.1.0

**Pull request merged**